scs-multiapi-gradle-plugin/build.gradle

```diff
@@ -20,7 +20,7 @@ repositories {
 }
 
 group = 'com.sngular'
-version = '5.1.1'
+version = '5.1.2'
```

Comment:
Being and enhancement we usually move to c.y+1.z

Reply:
Ok I will change. but your branch has the version 5.2.1

Reply:
Yeah, is ok, first to merge win, second should change the number. So if we merge this one first I should change the number too.